docs/synapse/userguides/storm_ref_cmd.rstorm

```diff
@@ -2468,6 +2468,35 @@ view.exec
 
 The ``view.exec`` command executes a Storm query in the specified view.
 
+**Behavior and Limitations**
+
+The ``view.exec`` command creates its own sub-runtime to execute a storm query in a different view.
+This sub-runtime creates a firm separation boundary between the source view and the destination
+view. This means nodes do not pass in or out across a ``view.exec`` boundary. Pipelines, events,
+messages, etc will NOT pass from the view.exec sub-runtime to the parent runtime or vice-versa. This
+includes ``$lib.print(...)``, ``$lib.warn(...)``, and other functions that may print to the CLI.
+
+It is possible for variables from the source runtime to pass into the sub-runtime created by
+``view.exec``. Variables from the source runtime may also be assigned to in the destination view.
+The interactive help example demonstrates this behavior:
+
+::
+
+  // Move some tagged nodes to another view
+  inet:fqdn#foo.bar $fqdn=$node.value() | view.exec 95d5f31f0fb414d2b00069d3b1ee64c6 { [ inet:fqdn=$fqdn ] }
+
+Here we have ``inet:fqdn`` nodes with the tag ``#foo.bar`` being lifted and their value (not the
+node) is saved into the ``$fqdn`` variable. This variable is later accessible in the ``view.exec``
+sub-runtime and used to create an ``inet:fqdn`` node in the destination view. If more than one
+``inet:fqdn`` node with the tag ``#foo.bar`` exists, the ``view.exec`` command would be executed
+once for each node in the pipeline as expected. Again, the actual nodes would not enter the
+sub-runtime created by the ``view.exec`` command though. Conversely, the sub-query executed in the
```

Comment:
The sentence that starts "Conversely..." is a bit confusing to me on a few counts. The word "conversely" implies an either/or to me - so my brain is trying to read this para as "either the var $fqdn is passed to the sub-runtime to create nodes OR some var assignment in the sub-runtime gets passed back to the primary runtime/view". Also, the situation you mention - a var assigned in the sub-runtime subquery being passed to the source view - does not appear to be happening here. Maybe a second example to illustrate that process? Unless I'm missing something? (I do like this example as it's a super simple query that shows the concept nicely.)

Reply:
Looks good - thanks for the clarifications!